packages/it-flat-batch/README.md

```diff
@@ -2,9 +2,10 @@
 
 [![Build status](https://travis-ci.org/achingbrain/it.svg?branch=master)](https://travis-ci.org/achingbrain/it?branch=master) [![Coverage Status](https://coveralls.io/repos/github/achingbrain/it/badge.svg?branch=master)](https://coveralls.io/github/achingbrain/it?branch=master) [![Dependencies Status](https://david-dm.org/achingbrain/it/status.svg?path=packages/it-flat-batch)](https://david-dm.org/achingbrain/it?path=packages/it-flat-batch)
 
-> Takes an async iterator that emits variable length arrays and emits them as fixed-size batches
+> Takes an (async) iterable that emits variable length arrays of things and
+> returns an async iterable that emits those thnigs in fixed-size batches.
```

Comment:
Can be on one line

Reply:
Fixed by ebabad16f043a6b3fdc0dbcf44e008510fcac282